__tests__/VueTributeTest.js

```diff
@@ -17,7 +17,7 @@ describe('VueTribute', () => {
       },
       propsData: testData
     })
-    expect(wrapper.name()).toBe('vue-tribute')
+    expect(wrapper.name()).toEqual('vue-tribute')
```

Comment:
This should stay as `toBe`, `toEqual` is for deep comparison.

Reply:
Check [Jest API Guide](https://jestjs.io/docs/en/expect#tobevalue), then you will see it actually uses `Object.is` to check the exact equality. For two arrays, `Object.is([], [])` will be false.

Or some explanation at [StackOverFlow](https://stackoverflow.com/questions/45195025/what-is-the-difference-between-tobe-and-toequal-in-jest/45209588#45209588).

Reply:
Yup, but this is a simple string comparison, and does not need `toEqual`

Reply:
lol, palm my face, I ignored that line, but it seems you already changed it, do I need to re-submit pull request?